guides/pages.yml

```
@@ -249,6 +249,50 @@
   pages:
     - title: "Gérer les Dépendances"
       url: "index"
+- title: "Utiliser TypeScript"
+  url: "typescript"
+  pages:
+    - title: "Utiliser TypeScript avec Ember"
+      url: "index"
+    - title: "TypeScript: pour bien démarrer"
+      url: "getting-started"
+    - title: "TypeScript: concepts fondamentaux"
+      url: "core-concepts"
+      pages:
+        - title: "TypeScript et Ember"
+          url: "index"
+        - title: "TypeScript: invokables"
+          url: "invokables"
+        - title: "TypeScript: routing"
```

To reuse what's already used in this file.

```suggestion
        - title: "TypeScript: routage"
```